index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
+appears i.e. to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property is mandatory and
+the value "VerifiableCredential" must appear in the set of values. Whilst
+it is good practice to include one additional value depicting the unique
+subtype of this verifiableCredential, it is not forbidden to either omit
+additional type values or to include multiple type values in the array.
+Since many verifiers will request verifiableCredentials of a specific
+subtype, then omitting the subtype value may make it more difficult for
+verifiers to inform the holder which verifiableCredential they require.
+When a verifiableCredential is of multiple subtypes, then listing all of
+them in the type property is sensible. However, the usage of the
+<code>type</code> property in a JSON-LD representation of a credential
+conveys more than what it means in a JSON representation of a credential.
+With JSON-LD, the technology is not only describing the categorization of the
+set of claims, the technology is also conveying the structure of the sub-graph
+of the claims.
+        </p>
+
+        <p>
+The primary purpose of the <code>@context</code> property, from a JSON-LD
+perspective, is to convey the meaning of the data and term definitions of
+the data in a verifiable credential in a machine readable way. When
+utilizing only a JSON representation the <code>@context</code> provides
+little technical benefits, but remains mandatory. The <code>@context</code>
```

Comment:
```suggestion
When
utilizing only a JSON representation the <code>@context</code> provides
little technical benefits, but remains mandatory.
```

Not true, having @context in JSON let's you apply a JSON Schema to the property to ensure certain contexts you expect exist, and if so, apply a rigid JSON Schema to the rest of the object.

Reply:
>That is, if you use @context in JSON, you get global semantics. Without @context, you have no idea if the object you're receiving just looks like the data you're expecting... that is, there are no confirmable global semantics.

How can you be certain of the global semantics unless you're actually checking the fully formed IRIs? It reads to me like you're saying that because you can check the `@context` property with the JSON Schema then you can assume there's proper global semantics that exist even if you're not relying on them and only using JSON. Wouldn't you need to utilize JSON-LD to check the global semantics since you can't be certain the contexts haven't changed since issuance? In which case, I don't see what additional technical benefits being gained are since the `@context` property is just being used for JSON-LD, hence it's a JSON-LD representation.

Reply:
Done, ty for the reminder @TallTed.

Reply:
> It reads to me like you're saying that because you can check the `@context` property with the JSON Schema then you can assume there's proper global semantics that exist even if you're not relying on them and only using JSON.

Yes, that's exactly what I'm saying (for most of the common use cases)... JSON-LD was designed for that to be true (again, most of the time... as long as you check `@context` for exactly what you expect). :)

> Wouldn't you need to utilize JSON-LD to check the global semantics since you can't be certain the contexts haven't changed since issuance?

If you want to be 100% sure, yes... but again, it's a best practice that all contexts that are used in production shouldn't be loaded off of the Web, so the meaning is unlikely to change (again, in well-designed in production systems). If you are JSON-only, and refuse to use JSON-LD, this is about as good as you can get (which is better than just vanilla JSON -- not having any global semantics at all).

> I don't see what additional technical benefits being gained are since the `@context` property is just being used for JSON-LD, hence it's a JSON-LD representation.

Unfortunately, this continues to be misunderstood about JSON-LD... and there are important nuances that make "Just check `@context` for expected values not true when 1) people use experimental/development/changing contexts, and 2) people mix other contexts in that redefine terms that can be redefined (note, none of the VC terms can be redefined).

It's the "provides little technical benefits" that I'm taking issue with. There are large technical benefits, which, while not perfect, enable even JSON to have global semantics (by having a very loose coupling and depending on JSON-LD for those global semantics). In other words, because someone went to the trouble of establishing global semantics properly via JSON-LD, JSON-only developers can depend on that w/o doing any JSON-LD processing (when using certain well defined, popular, and production-ready JSON-LD Contexts).

Reply:
> Done, ty for the reminder @TallTed.

Thanks, @msporny.

@kdenhartog -- There are two more unfenced instances of `@context`, in the segment you quoted from @msporny in https://github.com/w3c/vc-data-model/pull/847#discussion_r800107672

Reply:
> It's the "provides little technical benefits" that I'm taking issue with. There are large technical benefits, which, while not perfect, enable even JSON to have global semantics (by having a very loose coupling and depending on JSON-LD for those global semantics). In other words, because someone went to the trouble of establishing global semantics properly via JSON-LD, JSON-only developers can depend on that w/o doing any JSON-LD processing (when using certain well defined, popular, and production-ready JSON-LD Contexts).

This is a stretch I'd think, but since I see how it could paint JSON-LD in a negative light and in turn get weaponized if someone misses the nuance; I see why you wouldn't want it phrased the way it is currently. Can you provide some suggested language to replace it? I'm not going to die on this hill and happy to buy into your arguments.

I'm thinking something like:

> When utilizing only a JSON representation the <code>@context</code> provides some basic support of global semantics, but remains mandatory.

Reply:
> @kdenhartog -- There are two more unfenced instances of `@context`, in the segment you quoted from @msporny in [#847 (comment)](https://github.com/w3c/vc-data-model/pull/847#discussion_r800107672)

Thanks for the reminder - I missed those instances in the quotation. They're now properly updated.

Reply:
> @msporny @kdenhartog -- Please edit your comments here ([1](https://github.com/w3c/vc-data-model/pull/847#discussion_r800066968), [2](https://github.com/w3c/vc-data-model/pull/847#discussion_r800107672)), to codefence all instances of `@context` (that poor, endlessly tagged user)...

Yes, I bet he rues the day he chose that pseudonym !!

Reply:
> When utilizing only a JSON representation the `@context` provides some basic support of global semantics, but remains mandatory.

Yes, much better. How about:

> When encoding a pure JSON representation, the `@context` property remains mandatory and provides some basic support for global semantics.

Reply:
added in the PR now